pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumnodeconfigs.yaml

```
@@ -0,0 +1,112 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: ciliumnodeconfigs.cilium.io
+spec:
+  group: cilium.io
+  names:
+    categories:
+    - cilium
+    kind: CiliumNodeConfig
+    listKind: CiliumNodeConfigList
+    plural: ciliumnodeconfigs
+    singular: ciliumnodeconfig
+  scope: Namespaced
+  versions:
+  - name: v2alpha1
+    schema:
+      openAPIV3Schema:
+        description: "CiliumNodeConfig is a list of configuration key-value pairs.
+          It may optionally apply to only a set of nodes indicated by a label selector.
+          \n If multiple overrides apply to the same node, they will be ordered by
+          name with later Overrides overwriting any conflicting keys."
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Spec is the desired Cilium configuration overrides for a
+              given node
+            properties:
+              defaults:
+                additionalProperties:
+                  type: string
+                description: Defaults is treated the same as the cilium-config ConfigMap
+                  - a set of key-value pairs parsed by the agent and operator processes.
+                  Each key must be a valid config-map data field (i.e. a-z, A-Z, -,
+                  _, and .)
+                type: object
+              selector:
```

nit: `nodeSelector` would be a bit more consistent with other naming schemes we've used:
```suggestion
              nodeSelector:
```

---

I think the description got updated but not the field name?

---

no, I derped the json tag. Fix incoming.